core/modules/system/image.gd.inc

```diff
@@ -173,15 +158,31 @@ function image_gd_rotate(stdClass $image, $degrees, $background = NULL) {
  *   The width of the cropped area, in pixels.
  * @param $height
  *   The height of the cropped area, in pixels.
+ * @param $background
+ *   A string specifying an RGBA background color to use for the uncovered area
+ *   of the image after manipulation.  E.g. '#RGB', 'RGB', '#RGBA', 'RGBA',
+ *   '#RRGGBB', 'RRGGBB', '#RRGGBBAA', 'RRGGBBAA' or '0xRRGGBBAA'. For images
+ *   that support transparency, this will default to transparent. Otherwise it
+ *   will be white.
```

Comment:
It doesn't look like it, but let me get the tests passing, and see if it still works the same with hex.